src/components/dropdowns/primary-action-dropdown/primary-action-dropdown.js

```diff
@@ -54,7 +96,33 @@ const DropdownChevron = React.forwardRef((props, ref) => (
     onClick={props.onClick}
     isDisabled={props.isDisabled}
     isOpen={props.isOpen}
-    className={props.className}
+    css={[
+      css`
+        display: flex;
+        align-items: center;
+        padding: 0 ${vars['--spacing-4']};
+        border-radius: 0 ${vars['--border-radius-6']}
+          ${vars['--border-radius-6']} 0;
+        border-left: 1px solid ${vars['--color-gray']};
+        height: ${vars['--big-button-height']};
+      `,
+      props.isDisabled
```

Comment:
Sorry to be annoying but can we extract this maybe into a little helper. I find it easier to follow with e.g. an if/else there as this gets quite bulky.